docs/public-networks/reference/cli/options.md

```diff
@@ -2943,6 +2943,46 @@ random-peer-priority-enabled=true
 
 Enables or disables random prioritization of incoming connections. Enable in small, stable networks to prevent closed groups of peers forming. The default is `false`.
 
+### `receipt-compaction-enabled`
+
+<Tabs>
+
+<TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--receipt-compaction-enabled=<true|false>
+```
+
+</TabItem>
+
+<TabItem value="Example" label="Example">
+
+```bash
+--receipt-compaction-enabled=true
+```
+
+</TabItem>
+
+<TabItem value="Environment variable" label="Environment variable">
+
+```bash
+BESU_RECEIPT_COMPACTION_ENABLED=true
+```
+
+</TabItem>
+
+<TabItem value="Configuration file" label="Configuration file">
+
+```bash
+receipt-compaction-enabled=true
+```
+
+</TabItem>
+
+</Tabs>
+
+Enables or disables receipt compaction. Compacting receipts reduces storage by trimming unnecessary data from transaction receipts. Enabled by default, this option optimizes storage efficiency. It can be disabled if the compaction process introduces performance overhead.
```

Comment:
> Enabled by default

Currently it's disabled by default. Also, not released yet, but might be soon.

Reply:
> It can be disabled if the compaction process introduces performance overhead.

I'm not sure if this this performance overhead was actually observed but would defer to @jframe. Thinking we don't need to suggest why you might want to disable it, maybe could remove this sentence?

Reply:
Agree we should remove the sentence about performance. At the moment it's not clear without additional testing in high load environments.

Reply:
Hi @jframe and @siladu . I updated this PR. Is it ready for release?